Makefile

```diff
@@ -73,7 +73,7 @@ lint: setup
 	@$(VENV)/bin/flake8 --show-source \
 		--exclude=.venv,charmhelpers \
 		--filename *.py,20-functional.test \
-		hooks/ tests/ server/
+		hooks/* tests/ server/
```

Comment:
Yes, which is what is required for full coverage.  Should we plop a bash script in there we'll be screwed.